src/pages/standup/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { FC, memo } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+import { useRouter } from 'next/router';
+
+import { LOGIN_URL } from '@/constants/url';
+import useAuthenticated from '@/hooks/useAuthenticated';
+import StandUpContainer from '@/components/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+import { skipToken } from '@reduxjs/toolkit/dist/query';
+import PageNotFound from '../404';
+
+const StandUp: FC = memo(function StandUp() {
+    const { isLoggedIn, isLoading } = useAuthenticated();
+
+    const { isLoading: isAuthenticating } = useGetUserQuery(skipToken);
+    const router = useRouter();
+
+    const { dev } = router.query;
+
+    const handleConditionalRendering = () => {
+        if (!isAuthenticating && isLoggedIn) {
+            if (isLoading) {
+                return <p>Loading...</p>;
+            } else {
```

Comment:
if
else if 
else 
instead of writing like this

Reply:
fixed